platypus/platypus.lua

```diff
@@ -324,6 +346,11 @@ function M.create(config)
 		elseif state.current.wall_contact and platypus.allow_wall_jump then
 			state.current.wall_jump = true
 			state.previous.wall_jump = true
+			-- abort wall sliding when jumping from wall
```

Comment:
Maybe call platypus.abort_wall slide() instead? Or skip the conditional check and set current+previous wall_slide to false